compiled/web_bindings/index.html

```diff
@@ -3,17 +3,10 @@
   <head>
     <meta charset="utf-8" />
     <title>hello-snarkyjs</title>
-    <script type="module">
-      import { Field, isReady, SmartContract } from '/index.js';
-
-      class Something extends SmartContract {}
-
-      await isReady;
-
-      console.log(
-        `SnarkyJS initialized after ${performance.now().toFixed(2)}ms`
-      );
-      console.log(new Field(5));
+    <script type="importmap">
+      { "imports": { "snarkyjs": "./index.js" } }
+    </script>    
+    <script type="module" src="./simple_zkapp.js">
```

Comment:
better (manual) web test which not only loads snarkyjs, but also runs a smart contract happy path w/ proving